streampipes-platform-services/src/main/java/org/apache/streampipes/ps/PipelineElementTemplateResource.java

```diff
@@ -92,32 +95,35 @@ public ResponseEntity<?> getById(@Parameter(description = "The id of the pipelin
 
   @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
   @Operation(summary = "Store a new pipeline element template",
-      tags = {"Pipeline Element Templates"},
-      responses = {
-          @ApiResponse(responseCode = "200", description = "Template successfully stored")
-      })
+             tags = {"Pipeline Element Templates"},
+             responses = {
+                 @ApiResponse(responseCode = "200", description = "Template successfully stored")
+             })
   public ResponseEntity<Void> create(
       @RequestBody(description = "The pipeline element template to be stored",
-          content = @Content(schema = @Schema(implementation = PipelineElementTemplate.class)))
-      @org.springframework.web.bind.annotation.RequestBody PipelineElementTemplate entity) {
+                   content = @Content(schema = @Schema(implementation = PipelineElementTemplate.class)))
+      @org.springframework.web.bind.annotation.RequestBody PipelineElementTemplate entity
```

Comment:
I know it has already been there, but could you place replace this by an import statement? 🙂

Reply:
Ah okay, didn't see this
Does it make sense to keep both?

Reply:
I assume both are required. One is for spring's `RequestBody` and the other is for swagger.
Perhaps there is a way to harmonize this, but I would suggest doing this in another PR as it is independent of the changes from this PR.